From fa619132e14534707473485b54bff9945a3e11c9 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Fri, 8 Dec 2017 16:23:07 +0800
Subject: [PATCH 3019/5242] MLK-17126-5: arm64 dts: Move hdmi compatible
 srting

commit  c96af50027c78cfc6f96bdf3961d395268a2086e from
https://source.codeaurora.org/external/imx/linux-imx.git

Move imx8qm hdmi/dp compatible srting from soc specific
dts to board specific dts.

BuildInfo:
  - SCFW e0362348, IMX-MKIMAGE 9841373a, ATF e173337
  - U-Boot 2017.03-imx_v2017.03+g3535868

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts    |    1 +
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi      |    1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
index 937a716..5647d8ff 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-dp.dts
@@ -39,5 +39,6 @@
 };
 
 &hdmi {
+	compatible = "fsl,imx8qm-dp";
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 96ef9fd..6ceb7ff 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1234,7 +1234,6 @@
 	hdmi:hdmi@56268000 {
 		#address-cells = <1>;
 		#size-cells = <0>;
-		compatible = "fsl,imx8qm-dp";
 		reg = <0x0 0x56268000 0x0 0x100000>, /* HDP Controller */
 				<0x0 0x56261000 0x0 0x1000>; /* HDP SubSystem CSR  */
 		interrupts = <10 IRQ_TYPE_LEVEL_HIGH>;
-- 
1.7.9.5

